### PR TITLE
TK3.7: misc

### PR DESCRIPTION
**Pull request by frenchy64**
Related https://github.com/threatgrid/iroh/issues/3986

This PR contains many small fixes and improvements that IMO don't need a separate PR.

- add get-in-config to lib.metrics.{console,jmx,riemann} init!
- add get-in-config to wrap-request-logs
- cleaner ctia.bundle.routes ns form
- cleaner ctia.entity.event.store ns form
- add get-in-config to get-http-swagger
- add missing asset-related entries to PropertiesSchema
- remove top-level references to stores just for key set, replace with empty-stores
- improvements to ctia.stores.es.store
  - cleaner ns form
  - prevent accidental variable capture in def-es-store

<a name="qa">[§](#qa)</a> QA
============================

No QA is needed.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: TK3.7: misc
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

### CODE REVIEW

File: src/ctia/stores/es/store.clj [hunk 1]
```diff
@@ -17,6 +17,11 @@
    entity
    stored-schema
    partial-stored-schema]
+  (assert (symbol? store-name) (pr-str store-name))
+  (assert (keyword? entity) (pr-str entity))
+  (assert (symbol? stored-schema) (pr-str stored-schema))
+  (assert (not= 'state stored-schema) "Captured stored-schema binding!")
+  (assert (not= 'state partial-stored-schema) "Captured partial-stored-schema binding!")
```
Comment (frenchy64) [inline src/ctia/stores/es/store.clj:24]: Prevents accidental variable-capture.

File: src/ctia/properties.clj [hunk 2]
```diff
@@ -214,9 +214,15 @@
                       "ctia.migration.confirm?" s/Bool
                       "ctia.migration.restart?" s/Bool
 
+                      "ctia.store.asset" s/Str
+                      "ctia.store.asset-properties" s/Str
+                      "ctia.store.asset-mapping" s/Str
```
Comment (frenchy64) [inline src/ctia/properties.clj:219]: Missing from https://github.com/threatgrid/ctia/pull/936

File: src/ctia/properties.clj [hunk 3]
```diff
@@ -60,7 +60,7 @@
 
 (s/defschema StorePropertiesSchema
   "All entity store properties for every implementation"
-  (let [configurable-stores (map name (keys @store/stores))
+  (let [configurable-stores (map name (keys store/empty-stores))
```
Comment (frenchy64) [inline src/ctia/properties.clj:63]: Preferring `empty-stores` at top-level makes it possible to move `store/stores` into a TK service.

**Review by vemv — APPROVED** [review]
Sweet, no `sweet`!

**Pull request merged**